utils/crm_rpmcheck.py

```diff
@@ -31,6 +36,37 @@ def package_data(pkg):
 
     return {'name': pkg, 'error': "unknown package manager"}
 
+packages = None
```

Comment:
Private variables, which is not intended to be accessed directly from other modules, should start with `_`.